circle.yml

```diff
@@ -41,17 +41,6 @@ test:
     # behind a user namespace which Circle CI tests are run under
     - sudo env "PATH=$PATH" bin/yarn install --force
 
-deployment:
-  release:
-    tag: /v[0-9]+(\.[0-9]+)*/
-    owner: yarnpkg
-    commands:
-      # Only NPM is handled here - All other release files are handled in a webhook.
-      - echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" > ~/.npmrc
-      - ./scripts/set-installation-method.js "`pwd`/package.json" npm
-      # https://github.com/yarnpkg/yarn/issues/610, need to make yarn publish noninteractive
-      - npm publish --tag rc
```

Comment:
@bestander - Sure. This one is easy to move back into CircleCI if we want to do that. It's pretty straightforward.

@BYK The main reason was to make it consistent with all the other builds. We could continue publishing the npm build here, but the `.tar.gz`, `.msi` and `.js` builds will always be published on infra we run though, as they're all signed (Authenticode for .msi, GPG for everything else). There's no way I'd be comfortable with having a signing key sitting on CircleCI's infra 😃